src/Client.ts

```diff
@@ -101,6 +107,7 @@ export class Client extends Eris.Client implements ClientOptions {
 		// Object.assign(this, options); // eventually maybe we can just do this
 		this.prefix = options.prefix;
 		if (options.caseSensitivePrefix !== undefined) this.caseSensitivePrefix = options.caseSensitivePrefix;
+		if (options.caseSensitiveCommand !== undefined) this.caseSensitiveCommand = options.caseSensitiveCommand;
```

Comment:
I am a bit confused on why these if () checks are here like this? These options are all booleans and all have a default value set.

In the case of caseSensitiveCommand it is set to default false.

```js
if (options.caseSensitiveCommand) this.caseSensitiveCommand = options.caseSensitiveCommand
```
This if should only run when its true since its already defaulted to false. caseSensitiveCommand can only be true or false so having it check for undefined is unnecessary. If it is undefined it is already set to false. If it is false its already set to false. The only thing we need to check is `true`

Wait for Geo's thoughts on this before changing this though.

Reply:
This check is fine as it is for consistency with the others. The form you suggest is functionally identical in this case.